docs/src/app/components/pages/components/SelectField/ExampleDropDownMenuProps.js

```diff
@@ -0,0 +1,50 @@
+import React, {Component} from 'react';
+import SelectField from 'material-ui/SelectField';
+import MenuItem from 'material-ui/MenuItem';
+
+/**
+ * `SelectField` is implemented receiving the dropDownMenuProps object.
+ * This will allow to set any property of DropDownMenu that compose SelectField component.
+ */
+export default class SelectFieldExampleMenuClose extends Component {
+  state = {
+    value: 1,
+    closed: 0,
+    selected: 0,
+  };
+
+  handleChange = (event, index, value) => this.setState({value: value});
+
+  handleClose = () => this.setState({closed: this.state.closed++});
+
+  handleSelectionRenderer = (value) => this.setState({selected: value});
+
+  render() {
+    return (
+      <div>
+        <SelectField
+          floatingLabelText="Frequency"
+          value={this.state.value}
+          dropDownMenuProps={{
+            onchange: this.handleChange,
```

Comment:
Don't need to be onChange. I'll elaborate an example that handle every single props of DropDownMenu. 

I try to run the material-ui locally, but an error about multiple node versions is prevent this to happen. But I think the code is right.